distribution/helm/vector-aggregator/README.md

```diff
@@ -6,3 +6,13 @@ Our charts use Helm's dependency system, however we only use local dependencies.
 Head over to the repo for [more information on development and contribution](https://github.com/timberio/vector/tree/master/distribution/helm).
 
 To get started check out our [documentation](https://master.vector.dev/docs/setup/installation/platforms/kubernetes/)
+
+## Load Balancing (Beta)
+
+As of the 0.15.0 release we've included a HAProxy deployment to load balance traffic to Vector instances running in the [aggregator](https://vector.dev/docs/setup/deployment/roles/#aggregator) role.
+Using an external proxy allows us to scale aggregators horizontally in a `source` agnostic way, making it easy to migrate existing infrastructure to use Vector.
+
+To enable the HAProxy deployment you can adjust the `haproxy.enabled` value to `true`, by default this will create `frontend` and [server-template](http://cbonte.github.io/haproxy-dconv/2.4/configuration.html#4-server-template) configurations generated from your `service.ports` entries.
```

Comment:
Is it standard Kubernetes knowledge that when you say `haproxy.enabled`, you don't mean:

```
haproxy.enabled = true
```

you actually mean:

```
haproxy:
  enabled: true
```
?

I'm not saying I made this error or anything, of course, but I'm wondering if other people may?

Reply:
Actually, you can pass `--set haproxy.enabled=true` straight to helm. Is it worth giving examples of the helm command and how it would look in the values.yaml file? Or does everyone in Kubernetes land already know this?

Reply:
I think it's fairly common but I can adjust it to be the exact things to pass. Both are the same between dotting them together or nesting on a new line